bin/ci

```diff
@@ -7,16 +7,14 @@ declare main_branch="main"
 
 # Generate a coma-separated list of quoted app names that changed in this PR
 apps=$(
-  git whatchanged --name-only --pretty="" "origin/${main_branch}..${cursor}" | \
-  egrep "^apps" | \
-  sed "s/^apps\/\(.*\)\/.*$/\"\1\"/g" | \
+  git whatchanged --name-only --pretty="" "origin/${main_branch}..${cursor}" apps | \
+  awk -F '/' '{print "\"" $2 "\""}' | \
   sort -u | \
-  tr "\n" "," | \
-  sed "s/,$//"
+  paste -sd ','
 )
```

Comment:
Hat tip to @kernicPanel :mage:

Reply:
![](https://i.giphy.com/media/2xDc9oNdennpjgVTiv/giphy.webp)